apps/voting/contracts/Voting.sol

```diff
@@ -23,6 +23,7 @@ contract Voting is IForwarder, AragonApp {
     bytes32 public constant UNSAFELY_MODIFY_VOTE_TIME_ROLE = keccak256("UNSAFELY_MODIFY_VOTE_TIME_ROLE");
 
     uint64 public constant PCT_BASE = 10 ** 18; // 0% = 0; 1% = 10^16; 100% = 10^18
+    uint64 public constant MAX_HOLDERS_PER_MANAGER = 1024; // some reasonable number to mitigate unbound loop issue
```

Comment:
Isn't the "manager" term too vague for this context. I'd better go with the "delegate". It's well-known and election related term that brings much more context. We only need some clarification added that it's not a full fledged delegation everyone used to but the simple one.

Reply:
Agree here